plugins-mu/hm-stack/class-hm-stack-api.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * API Class to communicate with the HM Stack servers.
+ *
+ * @package HMStackIntegration`
+ */
+
+namespace HM_Stack;
+
+use WP_Error;
+
+/**
+ * Class HM_Stack_API
+ */
+class HM_Stack_API {
```

Comment:
Not totally sure why this is a class? There's no state here, a collection of functions would be simpler